### PR TITLE
Fix navigation support to re-use open editors for the same resource

### PR DESCRIPTION
**Pull request by mmews-n4**
see #1251

### CODE REVIEW

File: plugins/org.eclipse.n4js.ui/plugin.xml [hunk 1]
```diff
@@ -1307,8 +1307,8 @@ Contributors:
               </selectionEnablement>
               <editorInputEnablement>
                   <or>
-                 <instanceof value="org.eclipse.ui.IFileEditorInput"/>
-                 <instanceof value="org.eclipse.xtext.ui.editor.XtextReadonlyEditorInput"/>
+	                 <instanceof value="org.eclipse.ui.IFileEditorInput"/>
```
Comment (qtran-n4) [inline plugins/org.eclipse.n4js.ui/plugin.xml]: Problem with tab/space

File: plugins/org.eclipse.n4js.ui/plugin.xml [hunk 2]
```diff
@@ -1307,8 +1307,8 @@ Contributors:
...
+	                 <instanceof value="org.eclipse.xtext.ui.editor.XtextReadonlyEditorInput"/>
```
Comment (qtran-n4) [inline plugins/org.eclipse.n4js.ui/plugin.xml]: whitespace/tab

File: plugins/org.eclipse.n4js.ui/src/org/eclipse/n4js/ui/navigator/internal/N4JSResourceLinkHelper.java [hunk 3]
```diff
@@ -54,9 +52,13 @@
 	@Inject
 	private ExternalLibraryWorkspace externalLibraryWorkspace;
 
+	@Inject
+	private LanguageSpecificURIEditorOpener languageSpecificURIEditorOpener;
+
 	@Inject
 	private N4JSProjectExplorerHelper helper;
 
+	// XXX: Obsolete when virtual nodes are removed
```
Comment (qtran-n4) [inline plugins/org.eclipse.n4js.ui/src/org/eclipse/n4js/ui/navigator/internal/N4JSResourceLinkHelper.java]: Remove this comment?

Comment (mmews-n4) [reply plugins/org.eclipse.n4js.ui/src/org/eclipse/n4js/ui/navigator/internal/N4JSResourceLinkHelper.java]: Thanks, I changed it into a todo for #1259

**Comment by mmews-n4** [comment]
Thank you!

**Pull request merged**